sycl/test/basic_tests/min_max_test.cpp

```diff
@@ -0,0 +1,12 @@
+// REQUIRES: windows
+// RUN: %clangxx -fsyntax-only -Xclang -verify %s -I %sycl_include
```

Comment:
It's not clear from the description how exactly "the conflict" is exposed.
If it's a compilation error, you can add either add `-fsycl -fsycl-device-only` to enable SYCL mode and make SYCL headers "system" header or ignore warnings and notes from the SYCL headers by adding ` -Xclang -verify-ignore-unexpected=note,warning` to the run command.